### DISCUSSION TITLE
Unable to install from Helm

### DISCUSSION THREAD

**Discussion by adeel-ansari** [apache/doris#40345] [A - General / Q&A]
Hi,

I'm trying to install doris using the doris helm charts.  

I'm trying the following command:
``helm search repo doris-repo``

But I'm getting the following error:
`no results found`

I tried going directly to https://[charts.selectdb.com](https://charts.selectdb.com/)/ but I'm seeing the following:

`<Error>
<Code>AccessDenied</Code>
<Message>The bucket you access does not belong to you.</Message>
<RequestId>66D747625BF222363639FCA2</RequestId>
<HostId>selectdb-charts.oss-cn-beijing.aliyuncs.com</HostId>
<EC>0003-00000905</EC>
<RecommendDoc>https://api.aliyun.com/troubleshoot?q=0003-00000905</RecommendDoc>
</Error>`

Is the helm chart available to use?

**Comment by catpineapple** [comment]
After confirming that the warehouse and domain name are in normal use, please confirm that the network is normal and follow the operation document. There are roughly the following steps:
1. helm repo add doris-repo https://charts.selectdb.com
2. helm repo update doris-repo
3. helm install operator doris-repo/doris-operator
...

**Reply by adeel-ansari** [reply]
I tried those steps, but getting the same error:

`root# helm repo add doris-repo https://charts.selectdb.com
"doris-repo" has been added to your repositories

root# helm repo update doris-repo
Hang tight while we grab the latest from your chart repositories...
...Successfully got an update from the "doris-repo" chart repository
Update Complete. ⎈Happy Helming!⎈

root# helm install operator doris-repo/doris-operator
Error: INSTALLATION FAILED: chart "doris-operator" matching  not found in doris-repo index. (try 'helm repo update'): no chart version found for doris-operator-
`

Also, when I'm trying to go to charts.selectdb.com, it doesn't seem accessible.  I'm seeing this error:

![image](https://github.com/user-attachments/assets/428f665a-f032-493b-8823-e1f6f8fee520)

Is there a permission issue?

**Comment by catpineapple** [comment]
The browser access status of https://charts.selectdb.com you reported is normal. Can you show me your helm valuse.yaml configuration? I guess the configuration there is incorrect.

**Comment by adeel-ansari** [comment]
Hi,

Thanks for your help.  This is giving an error even before I'm trying to install the helm chart.

I'm using the instructions here; https://artifacthub.io/packages/helm/doris/doris-operator

When I do this step: `helm search repo selectdb`,  I'm expecting to see the doris-operator and the doris chart.

![image](https://github.com/user-attachments/assets/e2bf5828-b4e7-4e2b-9fd3-c425a04d2ce8)

But I only get the following message:
![image](https://github.com/user-attachments/assets/44b8f205-a51e-4e77-873c-80bcca4bda8b)

**Comment by catpineapple** [comment]
you can try : wget https://charts.selectdb.com/doris-operator-1.6.1.tgz,   It looks like access to the domain name occasionally fails.I need to confirm this issue again...

**Reply by adeel-ansari** [reply]
Thanks for your help with this issue.  I think you and the team may have resolved the issue with the helm chart as I can see the charts now:

helm search repo doris-repo
NAME                            CHART VERSION   APP VERSION     DESCRIPTION
doris-repo/doris                1.6.0           2.1.2           Apache Doris is an easy-to-use, high-performanc...
doris-repo/doris-operator       1.6.0           1.6.0           Doris-operator for doris creates, configures an...
